agent-service/tests/roster_import/test_utils_parsing.py

```
@@ -0,0 +1,192 @@
+from agents.roster.services.roster_import.utils import (
+    parse_date,
+    parse_time,
+    parse_boolean,
+    parse_int,
+    normalize_employment_type,
+)
+
+class TestParseDateHelper:
+    """Tests for _parse_date() helper."""
+
+    def test_parse_iso_format(self, handler):
```

Ruff (0.14.14)

[warning] Unused method argument: `handler`

(ARG002)